### PR TITLE
SharePoint custom forms and automated business workflow

### PR DESCRIPTION
**Pull request by jackwi-alt**
Edited content to meet MSTP standards and swap out Microsoft Flow with Power Automate and Powerapps with Power Apps.

#### Category
- [ ] Content fix
- [ ] New article
- [x] Example checked item (*delete this line*)

> For the above list, an empty checkbox is [ ] as in <kbd>[</kbd><kbd>SPACE</kbd><kbd>]</kbd>. A checked checkbox is [x] with no space between the brackets. Use the `PREVIEW` tab at the top right to preview the rendering before submitting your issue.
> 
> _(DELETE THIS PARAGRAPH AFTER READING)_

#### Related issues:
- fixes #issuenumber
- partially #issuenumber
- mentioned in #issuenumber

> If this fixes (aka: closes) or references an issue, please reference it here. This helps maintaining the issue list as it will (1) link the PR to the issue & (2) automatically close the issue when this PR is merged in.
> 
> _(DELETE THIS PARAGRAPH AFTER READING)_

#### What's in this Pull Request?

> Please describe the changes in this PR. Sample description or details around bugs which are being fixed.
> 
> _(DELETE THIS PARAGRAPH AFTER READING)_

#### Guidance

> *Please update this PR information accordingly. We'll use this as part of our release notes in monthly communications.*
> 
> *Please target your PR to 'master' branch. Released documents are in `live` branch.*
> 
> _(DELETE THIS PARAGRAPH AFTER READING)_

### CODE REVIEW

File: docs/business-apps/introduction-to-sharepoint-business-process-integration.md [hunk 1]
```diff
@@ -8,39 +8,39 @@ localization_priority: Priority
 
 # SharePoint custom forms and automated business workflow
```
Comment (chakkaradeep) [inline docs/business-apps/introduction-to-sharepoint-business-process-integration.md]: @jackwi-alt thinking of it now, how about the title: "Business Apps and Business Process Automation" ? That keeps it generic to the feature.

File: docs/business-apps/introduction-to-sharepoint-business-process-integration.md [hunk 2]
```diff
@@ -8,39 +8,39 @@ localization_priority: Priority
...
+Creativity and teamwork are essential to transforming business processes. Most processes center on content – files and data. SharePoint offers tools to gather and manage data in lists and libraries.  
 
-Office 365 provides connections to a range of third-party systems with Microsoft Flow and PowerApps. Flow lets you easily automate business processes and PowerApps lets you create great forms and mobile apps based on SharePoint files and data. Both PowerApps and Flow feature intuitive visual designers, so anyone can craft forms and screens in an easy-to-use interface.
+Office 365 provides connections to a range of third-party systems with Power Automate and Power Apps. Power Automate lets you easily automate business processes while Power Apps lets you create exceptional forms and mobile apps based on SharePoint files and data. Both Power Automate and Power Apps feature intuitive visual designers, so anyone can build forms and screens in an easy-to-use interface.
 
-## Microsoft Flow
+## Power Automate
```
Comment (chakkaradeep) [inline docs/business-apps/introduction-to-sharepoint-business-process-integration.md]: If we change the title as suggested earlier, we could change the title of this section to "Business Process Automation".

File: docs/business-apps/introduction-to-sharepoint-business-process-integration.md [hunk 3]
```diff
@@ -8,39 +8,39 @@ localization_priority: Priority
...
 
-Whether you start with a template or begin from scratch, using Flow to create automation features is intuitive and easy to understand.
+Whether you start with a template or begin from scratch, using Power Automate to create automation features is intuitive and easy to understand.
 
-Cross-tool functionality seamlessly moves your data from one operation to another. Since SharePoint is a core piece of the Office 365 suite,your data easily integrates with tools like Excel or Power BI reporting. And since it uses an Excelinspired expression language, these tools talk with each other, using formula language you already recognize and understand to integrate your data from one function to the next.
+Cross-tool functionality seamlessly moves your data from one operation to another. With SharePoint being a core component of the Microsoft 365 suite, your data easily integrates with tools like Excel or Power BI reporting. And because it uses an Excel-inspired expression language, these tools talk with each other, using formula language you already recognize and understand to integrate your data from one function to the next.
 
-## Microsoft PowerApps
+## Power Apps
```
Comment (chakkaradeep) [inline docs/business-apps/introduction-to-sharepoint-business-process-integration.md]: If we change the title as suggested earlier, we could change the title of this section to "Building Business Apps".

File: docs/business-apps/introduction-to-sharepoint-business-process-integration.md (hunk 1 above)
Comment (jackwi-alt) [reply docs/business-apps/introduction-to-sharepoint-business-process-integration.md]: Change made. Please review and merge my PR into master, if approved.

File: docs/business-apps/introduction-to-sharepoint-business-process-integration.md (hunk 2 above)
Comment (jackwi-alt) [reply docs/business-apps/introduction-to-sharepoint-business-process-integration.md]: Change made. Please mere my branch jackwi-alt:patch2 into master if approved.

File: docs/business-apps/introduction-to-sharepoint-business-process-integration.md (hunk 3 above)
Comment (jackwi-alt) [reply docs/business-apps/introduction-to-sharepoint-business-process-integration.md]: Change made. If approved, merge #5528 into master from branch jackwi-alt

**Pull request merged**